bindings.ios.js

```diff
@@ -163,6 +163,9 @@ nobleBindings.disconnect = function(deviceUuid) {
   this.RNBLE.disconnect(toAppleUuid(deviceUuid));
 };
 
+nobleBindings.enable = function() {
+};
```

Comment:
>  Is there a dialog automatically prompted

Right now nothing, I believe, but it's reported in `noble.state`.

I've been handling this case myself and providing my own instructions, and I personally feel that's better than introducing additional complexity and UI to this library, but I'm also not a maintainer of this library so I'd defer to @jacobrosenthal and others on whether they think that's appropriate.